DQM/SiTrackerPhase2/python/Phase2TrackerMonitorDigi_cfi.py

```diff
@@ -61,15 +61,15 @@
                             switch = cms.bool(True)
                         ),
                         DigiChargeH = cms.PSet(
-                            Nbins = cms.int32(50),
+                            Nbins = cms.int32(16),
                             xmin   = cms.double(-0.5),
-                            xmax   = cms.double(49.5),
+                            xmax   = cms.double(15.5),
                             switch = cms.bool(True)
                         ), 
                         TotalNumberOfDigisPerLayerH = cms.PSet(
-                            Nbins = cms.int32(1000),
-                            xmin   = cms.double(-0.5),
-                            xmax   = cms.double(4999.5),
+                            Nbins = cms.int32(10000),
```

Comment:
Render cannot support such a large binning I am afraid

Reply:
Well, it actually does in this case, but there are other 2D plots which are not being displayed due to the large binning used

Reply:
@jfernan2 , being ignorant on the matter, can you shine some light on what's the maximum number of 2D bins that can be rendered correctly?

Reply:
It is not just a matter of binning but a memory limit: objects beyond 8-10MB produce trouble, specially in Online DQM with the ProtoBuf system which goes LS by LS to display the evolving plot.

In this particular case, you have:
DigiXY: TH2 with 1250x1250 bins -> 6MB
DigiRZ: TH2 with 3000x1250 bins -> 15MB
(ocupancies are worse since they are TProfiles...)
OccupXY: TH2Profile with 1250x1250 bins -> 35MB
OccupRZ: TH2Profile with 3000x1250 bins -> 90MB

so, GUI is only able to render DigiXY at this point...[3]

The JSROOT RAW version seems to be able to render 15MB things... [1] but not always and not stable [2]....
[1] https://cmsweb.cern.ch/dqm/dev/jsroot/index.htm?file=https://cmsweb.cern.ch/dqm/dev/data/browse/ROOT/RelVal/CMSSW_11_2_x/DQM_V0001_R000000001__RelVal_wf23234_0_pr__CMSSW_11_2_X-PRcmssw_31563-9567__DQMIO.root
[2] https://cmsweb.cern.ch/dqm/dev/jsroot/index.htm?file=https://cmsweb.cern.ch/dqm/dev/data/browse/ROOT/RelVal/CMSSW_11_2_x/DQM_V0001_R000000001__RelVal_wf23434_999_pr__CMSSW_11_2_X-PRcmssw_31563-9567__DQMIO.root
[3] https://tinyurl.com/y5w6mkns

Reply:
thx, though to be clear this comment:

> specially in Online DQM with the ProtoBuf system which goes LS by LS to display the evolving plot.

certainly does not apply here.
@gsaha009 I would suggest you to reduce all the 2D maps to 1000x1000 max. The charge map is not really interesting and I would keep that optional (for the standalone wf)

Reply:
Yes, I have reduced the binning of all the 2D histograms. Also, I am keeping 5000 bins (max range 25000) of `TotalNumberOfDigisPerLayer` histogram.